printer_mods/henrikssn/euler_extruder/README.md

```diff
@@ -0,0 +1,50 @@
+# Euler Extruder for Voron AfterBurner
+
+### Overview
+
+The Euler Extruder is the most lightweight extruder option for the Voron AfterBurner, coming in at just 139 grams (including all hardware). There is a mount available for Nema17 based toolhead boards, such as [TurboCAN] and [Huvud], or you can screw any of the IGUS/Generic drag chain mount there.
+
+Performance-wise it is the same as Sherpa Mini, which is very similar to Orbiter/Galileo. All these are [more performant than original Clockwork].
+
+[TurboCAN]: https://github.com/henrikssn/TurboCAN
+[Huvud]: https://github.com/bondus/KlipperToolboard
+[more performant than original Clockwork]:
+
+
+<img src="Images/0.png" height=486 width=567 />
+<img src="Images/1.jpg" height=400 width=533 />
+<img src="Images/2.jpg" height=400 width=533 />
+<img src="Images/3.jpg" height=400 width=533 />
+
+### Printed parts
+
+Print the `extruder_motor_plate.stl`, all other parts can be reused from your existing extruder, given that you have the latest version of clockwork (with the short latch).
+
+Additonally, one of:
+
+* `[a]_chain_adapter.stl`: For attaching IGUS/Generic chain mount.
+* `[a]_turbocan_mount.stl`: For attaching [TurboCAN] or [Huvud] toolhead board.
```

Comment:
Since you renamed the file to `[a]_toolhead_board_mount.stl`, please update this in the README here as well.

Reply:
Thanks, done